### ISSUE TITLE
[Fixed in GH-6052] Safari Firefox: Missing Visual Focus on My Account: Close Account

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4337]
![Screen Shot 2019-08-29 at 9 06 56 PM](https://user-images.githubusercontent.com/15113219/63986224-04c3d880-caa1-11e9-994f-765240d7c548.png)

**Comment by kacperknapik** [comment]
- Chrome: ✔ (note: button focus border is barely visible)
- Firefox: ✔ (note: button focus border is huge)
- Safari: ❓

**Comment by KateChuen** [comment]
Chrome: OK

Missing keyboard accessibility & visual focus on:
- Cancel button on Firefox
- Cancel & Close Account buttons on Safari

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-30 at 4 06 05 PM](https://user-images.githubusercontent.com/15113219/73490494-aade7c00-437a-11ea-82dc-755c8f42b95a.png)
![Screen Shot 2020-01-30 at 4 06 57 PM](https://user-images.githubusercontent.com/15113219/73490495-aade7c00-437a-11ea-94c2-a93fc893f9b8.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome ✅
- Firefox ✅
- Safari ✅
